### PR TITLE
[4.x] htaccess gzip info

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #38028  .

### Summary of Changes
adds a comment at the top of the file pointing to a label near the bottom

### Testing Instructions
code review only as its just a comment

### CODE REVIEW

File: htaccess.txt [hunk 1]
```diff
@@ -16,6 +16,12 @@
 # set it here.
 ##
 
+## MISSING CSS OR JAVASCRIPT ERRORS
+#
+# If your site looks strange after enabling this, then your server is probably already
```
Comment (richard67) [inline htaccess.txt]:
```suggestion
# If your site looks strange after enabling this file, then your server is probably already
```
Maybe with this change it's more clear that "this" refers to the file and not to any of the stuff below?

**Comment by nikosdion** [comment]
I have tested this item :white_check_mark: successfully on b8f781b927e022699bc93d34ac7332b61f0325e9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38084">issues.joomla.org/tracker/joomla-cms/38084</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on b8f781b927e022699bc93d34ac7332b61f0325e9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38084">issues.joomla.org/tracker/joomla-cms/38084</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38084">issues.joomla.org/tracker/joomla-cms/38084</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**